ui/src/app/pages/workspace/workspace-library.tsx

```diff
@@ -40,17 +39,40 @@ const styles = reactStyles({
   },
   menuLinkSelected: {
     backgroundColor: colorWithWhiteness(colors.primary, 0.95)
+  },
+  libraryTabDivider: {
+    width: '100%',
+    margin: '0.5rem 0',
+    backgroundColor: colorWithWhiteness(colors.dark, .5),
+    border: '0 none',
+    height: 1
   }
 });
 
 const libraryTabEnums = {
-  FEATURED_WORKSPACES: {
-    title: 'Featured Workspaces',
-    icon: 'star'
-  },
   PUBLISHED_WORKSPACES: {
     title: 'Published Workspaces',
-    icon: 'library'
+    icon: 'bookmark',
+    filter: (publishedWorkspaces: WorkspacePermissions[], featuredWorkspaces: FeaturedWorkspace[]) => {
+      return publishedWorkspaces.filter(ws => !featuredWorkspaces.find(fws =>
+        ws.workspace.id === fws.id && ws.workspace.namespace === fws.namespace));
+    }
+  },
+  PHENOTYPE_LIBRARY: {
+    title: 'Phenotype Library',
+    icon: 'dna',
+    filter: (publishedWorkspaces: WorkspacePermissions[], featuredWorkspaces: FeaturedWorkspace[]) => {
+      return publishedWorkspaces.filter(ws => !!featuredWorkspaces.find(fws =>
+        ws.workspace.id === fws.id && ws.workspace.namespace === fws.namespace && fws.category === 'PHENOTYPE_LIBRARY'));
```

Comment:
Opt: Raw strings that are really enums is a tiny red flag. If you define the enum as a named entity in Swagger, you can probably refer to that enum value here. Not a huge deal, just a minor convenience / typo safety.

Reply:
Agreed. I had it as a string at first because I was toying with the idea of generated categories dynamically with new configs but this is better since I didn't follow through with that idea.

Reply:
I bet we have the same issues with impoverished generated enums here as on the backend.